saleor/graphql/account/types.py

```diff
@@ -91,12 +96,24 @@ class AddressInput(BaseInputObjectType):
             "[libphonenumber](https://github.com/google/libphonenumber) library."
         )
     )
-
     metadata = graphene.List(
         graphene.NonNull(MetadataInput),
         description="Address public metadata." + ADDED_IN_315,
         required=False,
     )
+    skip_validation = graphene.Boolean(
+        description=(
+            "Determine if the address should be validated. "
+            "By default, Saleor accepts only address inputs matching ruleset from "
+            "[Google Address Data]{https://chromium-i18n.appspot.com/ssl-address), "
+            "using [i18naddress](https://github.com/mirumee/google-i18n-address) "
+            "library."
```

Comment:
I thing that would be great to have here an additional hint, that for using the field, additional permission could be required, and each mutation that allows you to skip mutation will have more info about required permission. 
WDYT?

Reply:
I enriched the description as suggested.